scipy/stats/_stats_py.py

```diff
@@ -7042,8 +7047,7 @@ def ttest_ind_from_stats(mean1, std1, nobs1, mean2, std2, nobs2,
 
                          Sample   Sample
                    Size   Mean   Variance
-        Sample 1    13    15.0     87.5
-        Sample 2    11    12.0     39.0
+        Sample 1    13    15.0     87.5 Sample 2    11    12.0     39.0
```

Comment:
```suggestion
        Sample 1    13    15.0     87.5
        Sample 2    11    12.0     39.0
```

Reply:
Removed that change in https://github.com/scipy/scipy/commit/8b60b333ccf2df714169909440e9a8273395f5e9.